packages/kbn-apm-synthtrace/README.md

```diff
@@ -37,7 +37,7 @@ const traceEvents = timerange(from, to)
   .rate(10)
   .flatMap((timestamp) =>
     instance
-      .transaction('GET /api/product/list')
+      .transaction({transactionName: 'GET /api/product/list'})
```

Comment:
We could consider overloads here, since the second argument (`transactionType`) is optional and often left out.